src/level_four/core/controller/initial_device_provision_contoller.c

```diff
@@ -201,7 +203,7 @@ void device_provision_controller(){
             0xFF, 0xFF, //slot unlocked
             0x0E, 0x61, //ChipOptions
             0x00, 0x00, 0x00, 0x00, //certificate formatting disabled
-            0x53, 0x00, 0x53, 0x00, 0x73, 0x00, 0x73, 0x00, 0x73, 0x00, 0x38, 0x00, 0x7C, 0x00, 0x1C, 0x00, 0x3C, 0x00, 0x1A, 0x00, 0x1C, 0x00, 0x10, 0x00, 0x1C, 0x00, 0x30, 0x00, 0x12, 0x00, 0x30, 0x00
+            0x73, 0x00, 0x73, 0x00, 0x73, 0x00, 0x73, 0x00, 0x73, 0x00, 0x18, 0x00, 0x7C, 0x00, 0x3C, 0x00, 0x3C, 0x00, 0x3A, 0x00, 0x3C, 0x00, 0x30, 0x00, 0x3C, 0x00, 0x30, 0x00, 0x32, 0x00, 0x30, 0x00
```

Comment:
We should patch this on devices running older version of firmware.

Reply:
Also, check logic in line #146 (of function `check_provision_status`),
`if(cfg[86]==0x00 || cfg[87]==0x00){ //config zone and data zones are locked`

It should be && as written in the comment as per my understanding.

Reply:
2nd issue resolved and pushed

Reply:
https://app.clickup.com/t/37308523/CHI-2086

ticket raised